### PR TITLE
Pointed mailchimp to the proper mailing list

### PR DESCRIPTION
**Pull request by AndrewBestbier**
### Motivation
Pointing our mailchimp to the live mailing list, not a test mailing list.

### Test plan

Sign up and verify that mailchimp sign ups appear on the live mailing list.

### Pre-merge checklist

- [x] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
- [ ] Tester approved

### CODE REVIEW

File: services/mailchimp/sign-up/index.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import { mailchimpApi, formatFormInput, formatSignUpResponse } from '../utilities';
 
+const mailingListId = process.env.MAILING_LIST_ID;
```
Comment (lithin) [inline services/mailchimp/sign-up/index.js]: is this going to change with environment?

File: services/mailchimp/update-user/index.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import md5 from 'md5';
 import { mailchimpApi, formatUpdateResponse, formatFormInput } from '../utilities';
 
+const mailingListId = process.env.MAILING_LIST_ID;
```
Comment (lithin) [inline services/mailchimp/update-user/index.js]: move this inside the function to make sure it's loaded properly

File: services/mailchimp/utilities/index.js [hunk 3]
```diff
@@ -82,8 +82,8 @@ export function formatFormInput(event, isEmailEncrypted) {
     email_address: emailAddress,
     status: 'subscribed',
     merge_fields: {
-      FNAME: event.body.name || '',
-      LNAME: event.body.surname || '',
+      FIRSTNAME: event.body.name || '',
+      LASTNAME: event.body.surname || '',
```
Comment (lithin) [inline services/mailchimp/utilities/index.js]: 👍 for this ;))

File: services/mailchimp/utilities/test.js [hunk 4]
```diff
@@ -42,15 +42,15 @@ describe('formatSignUpResponse', () => {
 });
 
 describe('formatUpdateResponse', () => {
-  // if (res.last_changed !== res.timestamp_opt && data.merge_fields.FNAME !== '') {
+  // if (res.last_changed !== res.timestamp_opt && data.merge_fields.FIRSTNAME !== '') {
```
Comment (lpil) [inline services/mailchimp/utilities/test.js]: Commented out line

**Pull request merged**